src/test/java/jewellery/inventory/integration/UserCrudIntegrationTest.java

```diff
@@ -211,7 +211,7 @@ void updateUserFailsWhenNameDuplicate() {
             requestUpdate,
             UserResponseDto.class);
 
-    assertEquals(HttpStatus.BAD_REQUEST, responseEntity.getStatusCode());
+    assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
```

Comment:
This is a bad "fix" for a test. The requirements have changed and this is no longer a valid test case altogether. We should remove it.

Reply:
My bad. I did not intend to leave this 'fix' here. The test was supposed to be removed (already removed 3 other tests) this one I 'fixed' instead of commenting it out and then i forgot to remove it 🗡️